DS13/code/modules/overmap/overmap.dm

```diff
@@ -39,6 +39,9 @@ GLOBAL_LIST_INIT(overmap_ships, list())
 	else
 		. = ..()
 
+/obj/structure/overmap/can_be_pulled(mob/user) //Hahahah NO
```

Comment:
Anchored objects already can't be pulled,no need to override the proc unless it's dynamic behaviour.